benchmarks/initialize_submodule.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/sh
+
+function change_caliper_pom() {
+	local TEMPFILE=`tempfile`
+
+	one_jar_xml_script='		<plugin>
```

Comment:
If we are using Maven anyway, can you instead change the Java benchmark to use maven to build instead relying on manual invoke of javac? You can put all Java related sources into a java subdirectory and add a proper pom.xml to build a deploy jar without messing with caliper's pom.xml using shell script.

Reply:
Done